db/tasks.php

```diff
@@ -53,6 +53,15 @@
         'dayofweek' => '*',
         'month'     => '*'
     ),
+    array(
+        'classname' => 'tool_objectfs\task\delete_local_empty_directories',
+        'blocking'  => 0,
+        'minute'    => '*',
```

Comment:
I suspect this task will take probably 10 minutes to run, I'd lean more towards running this once a day